website/docs/r/pi_instance.html.markdown

```diff
@@ -99,7 +99,7 @@ Review the argument references that you can specify for your resource.
 - `pi_user_data` - (Optional, String) The base64 encoded form of the user data `cloud-init` to pass to the instance during creation. 
 - `pi_virtual_cores_assigned`  - (Optional, Integer) Specify the number of virtual cores to be assigned.
 - `pi_volume_ids` - (Optional, List of String) The list of volume IDs that you want to attach to the instance during creation.
-
+- `pi_deployment_type` - (Optional, String) Custom deployment type information (For Internal Use Only).
```

Comment:
I believe they like the markdown arguments to be listed in alphabetical order so this would need to  be moved up in the list.